src/arguments.cpp

```diff
@@ -271,6 +271,9 @@ Error Arguments::parse(const char* args) {
 
             CASE("fdtransfer")
                 _fdtransfer = true;
+                if (value == NULL) {
+                    msg ="fdtransfer path must not be empty";
```

Comment:
missing space after `=`

I'd also add `if (value == NULL || value[0] == 0)`

Reply:
Hmm I compared it to `safemode` which doesn't do the check. Why do some have `value[0] == 0` and some do not?

Anyway, changed